pkg/collector/corechecks/snmp/metadata/resource.go

```diff
@@ -0,0 +1,15 @@
+package metadata
+
+const (
+	ifNameOID = "1.3.6.1.2.1.31.1.1.1.1"
```

Comment:
This OID is already present in `oids.go`: https://github.com/DataDog/datadog-agent/blob/bd8ccbf42aeae2a4e7d6e4c2198d2c054c8b7479/pkg/collector/corechecks/snmp/metadata/oids.go#L22